docs/getting-started/dev-portal/introduction.md

```diff
@@ -1,18 +1,24 @@
 ## Introduction to Elements Dev Portal
 
-Elements comes in two flavours, the basic package which only covers a single API, and Elements Dev Portal, which is backed by Stoplight Platform and can do many many things. 
+Elements Dev Portal is backed by the Stoplight Platform and can do many many things. 
 
 - Support multiple APIs
 - Markdown Articles
 - [Hosted Mock Server](https://meta.stoplight.io/docs/platform/3.-design/d.setting-up-a-mock-server.md) Integration
 - Version Selector (Multiple Branches)
 - Powerful Search
 
-> The Dev Portal can only be tested on local host using the free plan. Please upgrade to starter or above to use it in a production environment.
+> Elements Dev Portal can be used on your local machine for free. If you need to use it in a production environment, please [subscribe to a paid plan](https://stoplight.io/pricing).
```

Comment:
Perhaps clarify the use case here (free during development / to try it out)? I could see folks reading this sentence and thinking that they can use dev portal offline / locally.

Reply:
What do you think? bdd9ff54b1a2434dc52989bae4f17c194597fc8d